### ISSUE TITLE
Error occurs accessing userEmail in email popups upon card creation

### ISSUE THREAD

**Issue by EHLOVader** [tungnguyenson/gmail-to-trello-extension#8]
When the email window is a popup (ie minimal UI) the logic used to grab the users email is not working and results in an error on card creation.

Related to this commit: [38315703b8867ce1ab06843612fc062e94b86e35](https://github.com/tungbi/gmail-to-trello-extension/commit/38315703b8867ce1ab06843612fc062e94b86e35)

Specifcally [this](https://github.com/tungbi/gmail-to-trello-extension/blob/master/chrome/model.js#L194) line 

Here is the exact error message received in the console:

```
 Uncaught TypeError: Cannot read property 'replace' of undefined
    GmailToTrello.Model.submit      chrome-extension://oceoildfbiaeclndnjknjpfaoofeekgl/model.js:194 
    (anonymous function)            chrome-extension://oceoildfbiaeclndnjknjpfaoofeekgl/app.js:79
    EventTarget.fire                chrome-extension://oceoildfbiaeclndnjknjpfaoofeekgl/lib/eventTarget.js:35 
    (anonymous function)            chrome-extension://oceoildfbiaeclndnjknjpfaoofeekgl/views/popupView.js:230 
    x.event.dispatch                chrome-extension://oceoildfbiaeclndnjknjpfaoofeekgl/lib/jquery.min.js:5 
    y.handle                        chrome-extension://oceoildfbiaeclndnjknjpfaoofeekgl/lib/jquery.min.js:5
```

**Comment by EHLOVader** [comment]
Actually discovered the error starts from the GLOBALS in the popup view. It is undefined.

**Comment by acoven** [comment]
Fixed with #18.
